src/storage/storage-cache.ts

```diff
@@ -238,8 +243,12 @@ export class StorageCache {
 
     // Assume this is accepted and latest for the moment.
     return {
-      docIngested: signedDoc,
-      ingestResult: IngestResult.AcceptedAndLatest,
+        kind: 'success',
+        maxLocalIndex: this._storage.storageDriver.getMaxLocalIndex(),
+        doc: signedDoc,  // this is missing _localIndex for now
+        docIsLatest: true,
+        prevDocFromSameAuthor: null, // we don't actually know this
+        prevLatestDoc: null, // we don't actually know this
```

Comment:
Hmm... `StorageCache` doesn't have the richer context to fill out the IngestEvent properly, so I had to just set some values to null.  (See comments)